pyproject.toml

```
@@ -0,0 +1,82 @@
+[build-system]
+# Defined by PEP 518
+requires = [
+    "setuptools>=64",
+    "wheel",
+]
+# Defined by PEP 517
+build-backend = "setuptools.build_meta"
+
+[project]
+authors = [
+    {name = "Iris-Grib Contributors", email = "scitools.pug@gmail.com"}
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: MacOS",
+    "Operating System :: POSIX",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: Unix",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Atmospheric Science",
+    "Topic :: Scientific/Engineering :: Visualization",
+]
+dynamic = [
+    "dependencies",
+    "readme",
+    "version",
+]
+description = "Grib interface for Iris"
+keywords = [
+    "iris",
+    "grib",
+]
+license = {text = "BSD-3-Clause"}
+name = "scitools-iris"
+requires-python = ">=3.9"
+
+[project.urls]
+Code = "https://github.com/SciTools/iris-grib"
+Discussions = "https://github.com/SciTools/iris-grib/discussions"
+Documentation = "https://iris-grib.readthedocs.io/en/stable/"
+Issues = "https://github.com/SciTools/iris-grib/issues"
+
+[tool.setuptools]
+licence-files = ["LICENSE"]
```

```suggestion
license-files = ["LICENSE"]
```

American-English I guess